Plots/Contours/NCL_conwomap_3.py

```diff
@@ -0,0 +1,92 @@
+"""
+NCL_conwomap_3.py
+=================
+Concepts illustrated:
+  - Drawing a simple contour plot
+  - Generating dummy data using "random_normal"
+  - Masking mirrored contour data
+  - Drawing a perimeter around areas on a contour plot with missing data
+  - Turning off the bottom and right borders of a contour plot
+  - Using "getvalues" to retrieve resource values
+  - Changing the labels and tickmarks on a contour plot
+  - Adding a complex Greek character to a contour plot
+  - Moving the contour informational label into the plot
+  - Forcing tickmarks and labels to be drawn on the top X axis in a contour plot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/conwomap_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/conwomap_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Generate random data:
+
+xlist = np.linspace(0, 31.0, 31)
+ylist = np.linspace(0, 31.0, 31)
+xdata, ydata = np.meshgrid(xlist, ylist)
+
+zdata = np.random.normal(0, 3.0, size=(31, 31))
+
+###############################################################################
+# Create figure
+plt.figure(figsize=(10, 10))
+
+# Create axes
+ax = plt.axes()
+
+# Use geocat.viz.util convenience function to set axes limits & tick values without calling several matplotlib functions
+gvutil.set_axes_limits_and_ticks(ax, xlim=(0, 30), ylim=(0, 30), xticks=None, yticks=None, xticklabels=None, yticklabels=None)
+
+# Use geocat.viz.util to add major and minor tics
+gvutil.add_major_minor_ticks(ax, x_minor_per_major=4, y_minor_per_major=4)
+
+# Use geocat.viz.util convenience function to add titles to left and right of the plot axis.
+gvutil.set_titles_and_labels(ax, ylabel="wave number", labelfontsize=24)
+
+# Set ticks and labels only on left and top of plot
+ax.xaxis.tick_top()
+ax.yaxis.tick_left()
+
+# Make tick font size bigger
```

Comment:
Instead of these **Lines 57** through **59**, I would add an argument `labelsize=18` to the above function call `gvutil.add_major_minor_ticks()`, which handles the same task.